core/src/marine_core.rs

```diff
@@ -151,6 +169,26 @@ impl MarineCore {
         records.into()
     }
 
+    pub fn list_outdated(&self) -> Vec<OutdatedModule> {
+        self.modules
+            .iter()
+            .filter_map(|(name, module)| {
+                module
+                    .check_update_status(name.clone())
```

Comment:
I believe that it's better either to rewrite the signature of this function to receive `AsRef<str>` or to pass a reference here.

Reply:
ditto